source/components/JsxParser.js

```diff
@@ -89,6 +89,15 @@ export default class JsxParser extends Component {
           default:
             return undefined
         }
+      case 'UnaryExpression':
+        switch (expression.operator) {
+          case '+':
+            return expression.argument.value
+          case '-':
+            return -1 * expression.argument.value
+          default:
```

Comment:
I think we should also support the `!` operator, which also falls under `UnaryExpression` in Acorn.
```suggestion
          case '!':
            return !expression.argument.value
          default:
```

Reply:
This won't work and will fail the tests, you must also coerce T/F into strings to render and then back into T/F for props.  I will update with an additional commit to accommodate this feature request and pass all tests.

Reply:
Addressed in newest commit:  Update to include unary NOT operations support.

This suggested solution actually will not work, must instead coerce the T/F values to a string in order to be rendered, and then back to a T/F for props.